src/kibali/PermissionsStubGenerator.cs

```diff
@@ -26,23 +24,33 @@ public string GenerateTable()
         authZChecker.Load(this.document);
 
         var resource = authZChecker.FindResource(this.path);
+        var table = this.UnsupportedPermissionsStub();
```

Comment:
Can we add a boolean parameter to determine whether to generate a default permissions table or not? When updating existing docs I don't want to overwrite an existing table with an empty one in case permissions are not found. I would expect a null or an empty string which will signal me to leave the current table as is.

Reply:
Done.